tools/salmon/macros.xml

```diff
@@ -1,5 +1,5 @@
 <macros>
-    <token name="@VERSION@">1.3.0</token>
+    <token name="@VERSION@">1.4.0</token>
```

Comment:
@gallardoalba turns out the update is not that complicated. Most tests have passed.